sdk/core/azure-core-amqp/samples/internal/eventhub_sas_reader_sample/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ cmake_minimum_required(VERSION 3.13)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+if(MSVC)
+  add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
```

Comment:
Is this due to our code or a dependency?  It would be nice to not need to have this disabled and to not depend on non-secure versions.